mods/ra/rules/ai.yaml

```diff
@@ -80,7 +80,7 @@ Player:
 		RefineryTypes: proc
 	BaseBuilderBotModule@rush:
 		RequiresCondition: enable-rush-ai
-		MinimumExcessPower: 60
+		MinimumExcessPower: 0
```

Comment:
Won't the AI build regular defences and cause itself to go on low power?

Reply:
I think 4 building is OK to trigger a power increasement?

Changed to 3 building to trigger a power increasement.

Reply:
By introducing this module, I tend to make AI can maintain less powerplant at early game, give them chance for better eco.

Reply:
I actually suggest change bot's BO, add a delay to basic defence, as they are useless at very early game